.travis.yml

```diff
@@ -7,6 +7,15 @@ os: osx
 # on Traivs.
 env:
   - TEST_DIR=com.google.code.gson-gson
+  - TEST_DIR=org.joda-joda-convert
+  - TEST_DIR=joda-time-joda-time
```

Comment:
I mean it is definitely correct:
https://github.com/JodaOrg/joda-time/blob/master/pom.xml#L8

just odd